scripts/add_nagim_gvcfs_to_tob_wgs_test_metamist.py

```diff
@@ -0,0 +1,143 @@
+from metamist.graphql import gql, query
+from metamist.apis import ParticipantApi, AnalysisApi
+from metamist.models import (
+    ParticipantUpsert,
+    SampleUpsert,
+    SequencingGroupUpsert,
+    AssayUpsert,
+    AnalysisStatus,
+    Analysis,
+)
+from cpg_utils import to_path
+import csv
+
+import click
+
+PARTICIPANT_QUERY = gql(
+    """
+    query ($project: String!) {
+        project(name: $project) {
+            id
+            participants {
+            externalId
+            id
+            samples {
+                id
+                externalId
+                sequencingGroups {
+                externalIds
+                id
+                meta
+                platform
+                technology
+                type
+                assays {
+                    meta
+                    id
+                    }
+                }
+            }
+        }
+    }
+}
+"""
+)
+
+
+@click.command()
+@click.option(
+    '--project',
+    required=True,
+    help='The name of the project to add samples to.',
+)
+@click.option(
+    '--project_id',
+    required=True,
+    type=int,
+    help='The ID of the project to add samples to.',
+)
+@click.option(
+    '--new_samples_path',
+    required=True,
+    help='The path to the file containing the newly created samples.',
+)
+def main(project: str, projectID: int, new_samples_path: str):
+    # Read the CSV file into a dictionary
+    extID_to_row = {}
```

Comment:
Python variable names are snake case (no capitals except for constants). Can you fix these up below?